app/main.py

```
@@ -1,2 +1,62 @@
+from typing import Any
+
+
+class Node:
+    def __init__(self, key: Any, value: Any):
+        self.key = key
+        self.value = value
+        self.hash = hash(key)
+
+
 class Dictionary:
-    pass
+    def __init__(self, load_factor: float = 2/3) -> None:
+        self.hash_table = [None] * 8
+        self.empty_cells = [cell for cell in range(8)]
+        self.load_factor = load_factor
+
+    def __setitem__(self, key, value) -> None:
+        self._resize()
+        new_node = Node(
+            key=key,
+            value=value
+        )
+        cell = self.get_slot(new_node.key)
+
+        if self.hash_table[cell] is None or self.hash_table[cell].key == new_node.key:
+            self.hash_table[cell] = new_node
+        else:
+            for cell in range(len(self.hash_table)):
+                if self.hash_table[cell] is None:
+                    self.hash_table[cell] = new_node
+
+    def __getitem__(self, key) -> Any:
+        if self.hash_table[self.get_slot(key)] is None:
+            raise KeyError
+        if self.hash_table[self.get_slot(key)].key == key:
+            return self.hash_table[self.get_slot(key)].value
+        else:
+            for cell in self.hash_table:
+                if not isinstance(cell, Node) or cell.key != key:
+                    continue
+                elif cell.key == key:
+                    return cell.value
+            # raise KeyError
```

The linear probing in `__getitem__` has similar issues to `__setitem__`. If the key is not in the initial slot, it iterates through the entire hash table. This can lead to O(n) time complexity in the worst case.

Also, you commented out `raise KeyError`. You should raise `KeyError` if the key is not found.